cmd/new.go

```diff
@@ -766,17 +767,21 @@ func promptForConfig(
 
 // promptForValue prompts the user for a value with a defaultValue preselected. Hitting enter accepts the
 // default. If yes is true, defaultValue is returned without prompting. isValidFn is an optional parameter;
-// when specified, it will be run to validate that value entered. An invalid value will result in an error
-// message followed by another prompt for the value.
+// when specified, it will be run to validate that value entered. When this function returns a non empty string
+// validation is assumed to have failed and an error is printed. The string returned by isValidFn is also displayed
+// to provide information about why the validation failed. A period is appended to this message. `promptForValue` then
+// pompts again.
 func promptForValue(
-	yes bool, prompt string, defaultValue string, secret bool,
-	isValidFn func(value string) bool, opts display.Options) (string, error) {
+	yes bool, valueType string, defaultValue string, secret bool,
+	isValidFn func(value string) string, opts display.Options) (string, error) {
 
 	if yes {
 		return defaultValue, nil
 	}
 
 	for {
+		var prompt string
+
 		if defaultValue == "" {
 			prompt = opts.Color.Colorize(
 				fmt.Sprintf("%s%s:%s ", colors.BrightCyan, prompt, colors.Reset))
```

Comment:
`prompt` => `valueType` here, right?

```suggestion
				fmt.Sprintf("%s%s:%s ", colors.BrightCyan, valueType, colors.Reset))
```

Reply:
Yup!  Thanks!